scripts/jenkins/ardana/ansible/group_vars/all/vcloud.yml

```diff
@@ -48,3 +49,10 @@ heat_stack_output_queries:
    stdout: "{{ heat_stack_output | default([]) | selectattr('output_key', 'equalto', 'controller-conf-ips') | map(attribute='output_value') | list | flatten }}"
  - item: "compute-conf-ips"
    stdout: "{{ heat_stack_output | default([]) | selectattr('output_key', 'equalto', 'compute-conf-ips') | map(attribute='output_value') | list | flatten }}"
+
+virtual_node_group_aliases:
+  controller: 'controller'
+  # need to use different group aliases, because Crowbar uses 'compute-kvm'
+  # instead of 'compute' in the crowbar batch scenario files
```

Comment:
This is not really true. At least not in general. The aliases are controlled with `want_node_aliases` variable in `mkcloud`. For this scenario it is set here: https://github.com/SUSE-Cloud/automation/blob/master/jenkins/ci.suse.de/templates/cloud-mkcloud-job-2nodes-template.yaml#L27

Reply:
True, but for now we only target the crowbar batch files used by the legacy Crowbar CI gating jobs. I updated the comment to include that information.